pkg/cli/cmd/cli.go

```diff
@@ -57,5 +69,26 @@ func CheckRunning(c *cc.CommonCtx) error {
 		}
 	}
 
-	return nil
+	return ErrIsRunning
+}
+
+func grpcAvaialable(grpcAddress string) bool {
+	tlsConf, err := grpcurl.ClientTLSConfig(true, "", "", "")
+	if err != nil {
+		return false
+	}
+
+	creds := credentials.NewTLS(tlsConf)
```

Comment:
I added that function to call directly the cmd.Host because the health endpoint configuration is optional so there might be cases where a topaz instance is running without a health endpoint.